src/universe/pc.cpp

```diff
@@ -299,7 +299,6 @@ void cPlayer::acid(int how_much) {
 
 void cPlayer::restore_sp(int amt) {
 	if(!is_alive()) return;
-	if(cur_sp >= max_sp) return;
```

Comment:
I think I already mentioned that this is incorrect and the proper fix is to call `drain_sp` from the special node.

I imagine the reason that wasn't already done is because `drain_sp` implements resistance, and the goal was that the special node can't be resisted, so I also added a flag to disable resistance.

See d7dcf2464426496cb64d34faa77dccabad251ed7 for details.

Reply:
Ok.